.pre-commit-config.yaml

```
@@ -40,3 +40,9 @@ repos:
         types: [python]
         pass_filenames: false # Run full test suite
         stages: [pre-commit] # Only run on commit
+  - repo: https://github.com/google/addlicense
+    rev: v1.1.1  # Use a specific release or commit SHA
+    hooks:
+    - id: addlicense
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)